test/controllers/CreateRecordStartControllerSpec.scala

```diff
@@ -52,7 +53,7 @@ class CreateRecordStartControllerSpec extends SpecBase {
 
         status(result) mustEqual SEE_OTHER
 
-        redirectLocation(result).value mustEqual routes.IndexController.onPageLoad.url
+        redirectLocation(result).value mustEqual routes.TraderReferenceController.onPageLoad(NormalMode).url
```

Comment:
Unit test name also needs to be updated

`"must redirect to the index controller page when the user click continue button"`

Reply:
Updated :D